src/Input/Select/Select.tsx

```diff
@@ -54,12 +56,30 @@ const Select: ISelect = (props: Props) => {
     [status]
   );
 
+  // warn non-string type value, defaultValue
+  React.useEffect(
+    function warnNonStringValueAndDefaultValue() {
+      const isValueNotString = value && typeof value !== 'string';
+      const isDefaultValueNotString =
+        defaultValue && typeof defaultValue !== 'string';
```

Comment:
I think we can just leave `typeof value !== 'string';` without `value &&`. It will be more intuitive to me.

Because I was thinking about whether it works when the value is an empty string or `true`.

Reply:
value and defaultValue are optional, so I think we should not warn when they are undefined. So maybe I make it 
```
typeof value !== 'undefined' && typeof value !== 'string';
```

WDYT?

Reply:
Yeah, that'd be more clear. 👍